mypyc/irbuild/match.py

```diff
@@ -331,7 +331,7 @@ def bind_as_pattern(self, value: Value, new_block: bool = False) -> None:
                 self.builder.goto(self.code_block)
 
     @contextmanager
-    def enter_subpattern(self, subject: Value) -> Generator[None, None, None]:
+    def enter_subpattern(self, subject: Value) -> Generator[None]:
```

Comment:
#18382 also adds `from __future__ import annotations. Including it here would just create a merge conflict and isn't strictly necessary anyway.